extensions/panache/hibernate-orm-panache-kotlin/runtime/src/main/java/io/quarkus/hibernate/orm/panache/kotlin/runtime/KotlinJpaOperations.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.hibernate.orm.panache.kotlin.runtime;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import javax.persistence.EntityManager;
+
+import io.quarkus.hibernate.orm.panache.common.runtime.AbstractJpaOperations;
+
+public class KotlinJpaOperations extends AbstractJpaOperations<PanacheQueryImpl<?>> {
```

Comment:
As a side note, we should really find a way for users to not use our __Operation__ classes.

Reply:
That would be lovely, for sure.  I've tried to document them as private but that's not really going to stop anyone.